rest-assured/pom.xml

```diff
@@ -49,8 +49,9 @@
                     <instructions>
                         <Export-Package>
                         	!io.restassured.path.xml.*,
-                        	!io.restassured.path.json.*,
-                        	!io.restassured.common.*,
+                            !io.restassured.path.json.*,
```

Comment:
@johanhaleby @azzazzel @ponziani hi, this is required to get the maven duplicate finder happy again, but the OSGi tests in this project still fail:

https://travis-ci.org/rest-assured/rest-assured/jobs/494305823

Is this the right thing to do? How do we get the OSGi tests fixed up?

Reply:
I'll look into it.

Reply:
I just sent a PR to you.
See https://github.com/markkolich/rest-assured/pull/1